.github/workflows/tag.yml

```diff
@@ -21,6 +21,10 @@ jobs:
         git config user.name 'Capa Bot'
         name=${{ github.event.release.tag_name }}
         git tag $name -m "https://github.com/mandiant/capa/releases/$name"
+    - name: Tag capa-rules major version
+      run: |
+        name-major="$(cut -d '.' -f1 <<< "$name")"
```

Comment:
alternative (using parameter extension)[https://developpaper.com/bash-technique-use-parameter-extension-to-remove-prefix-or-suffix-of-string-variable-value]: `name-major=${name%%.*}`

I am not sure if this proposal is clearer that the current one and it is not important, both work 😄